go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/omec-project/nas v1.3.0
 	github.com/omec-project/ngap v1.2.2
 	github.com/omec-project/nrf v1.4.1
```

Comment:
I think you need to run `go mod tidy` because this dependency (`nrf`) is not needed anymore
```suggestion
```